markets/spot-market/contracts/modules/token/AsyncOrderClaimTokenModule.sol

```diff
@@ -2,33 +2,11 @@
 pragma solidity >=0.8.11 <0.9.0;
 
 import "@synthetixio/core-modules/contracts/modules/NftModule.sol";
-import "../../../contracts/interfaces/IAsyncOrderClaimTokenModule.sol";
 
 /**
  * @title Module with custom NFT logic for the async order claim token
  * @dev See IAsyncOrderClaimToken.
  */
-contract AsyncOrderClaimTokenModule is IAsyncOrderClaimTokenModule, NftModule {
-    /**
-     * @inheritdoc IAsyncOrderClaimTokenModule
-     */
-    function mint(address owner) external returns (uint256 id) {
-        OwnableStorage.onlyOwner();
+contract AsyncOrderClaimTokenModule is NftModule {
 
-        id = totalSupply();
-        _mint(owner, id);
-
-        emit Mint(owner, id);
-    }
```

Comment:
this was throwing a compiler error; we can't override `mint` since `NFTModule` already is.  we would have to inherit directly from `erc721` then re-add all the initialization code.

instead, i opted to remove the special mint/burn functions here and just use the `totalSupply` as the id, the one place we mint in `AsyncOrderModule`.  burn didn't change so it was redundant.